src/zenml/cli/__init__.py

```diff
@@ -407,21 +407,29 @@
 `aws_session_token`). If you do not have a specific secret type you wish to use,
 ZenML will use the `arbitrary` type to store your key-value pairs.
 
-To register a secret, use the `register` command:
+To register a secret, use the `register` command and pass the key-value pairs
+as command line arguments:
 
 ```bash
-zenml secret register SECRET_NAME
+zenml secret register SECRET_NAME --key1=value1 --key2=value2 --key3=value3 ...
 ```
 
-If you wish to register a single secret non-interactively, you can
-pass in a key-value pair at the command line, as in the following example:
+Note that the keys and values will be preserved in your `bash_history` file, so
+you may prefer to use the interactive `register` command instead:
 
 ```shell
-zenml secret register SECRET_NAME -k KEY -v VALUE
+zenml secret register SECRET_NAME -i
+```
+
+As an alternative to the interactive mode, also useful for values that
+are long or contain newline or special characters, you can also use the special
+`@` syntax to indicate to ZenML that the value needs to be read from a file:
+
+```bash
+zenml secret register SECRET_NAME --attr_from_literal=value \
```

Comment:
In this case attr_from_literal is the key? Not 100% clear imo - i like hwo its explained in the register_secret cli docstring

Reply:
I'll update the example to be closer to what I have in the CLI docstring.